apb.yml

```diff
@@ -626,6 +626,14 @@ _apb_auth_role_mapper_replace_role: &_apb_auth_role_mapper_replace_role
   description: Whether to add to the current roles, or replace the current roles with the mapped ones. Replaces if set to true.
   type: boolean
   display_group: RoleMapping (Optional)
+# Smart Router props
+_apb_smartrouter_protocol: &_apb_smartrouter_protocol
+  name: apb_smartrouter_protocol
+  default: http
```

Comment:
Can this have a limitted set of values like "http"/"https"? If so, I recommend extending the existing anchor `_apb_protocol_type` that will render a dropdown and override the necessary values. If it has more options like "ws", you can also override the `enum` attribute